src/main/java/mate/academy/spring/model/User.java

```diff
@@ -16,6 +22,11 @@ public class User {
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
which default type for relation Many-to-Many?

Reply:
lazy, but i saw on hibernate topic that we can annotate it for comfortable view and work